File: inputs/simpleFunc.c
int add(int a, int b) {
    return a + b;
}

int add3(int a, int b, int c) {
    return a + b + c;
}

int add4(int a, int b, int c, int d) {
    return a + b + c + d;
}

int main() {
    int x = 2;
    int y = 3;
    int z = add(x, y);
    z = add3(x, y, z);
    z = add4(x, y, z, 10);
    return z;
}

File: inputs/simpleFunc.ml
i32 add3(i32 a, i32 b, i32 c) {
return a + b + c;
}

i32 main() {
i32 x = 2;
i32 y = 3;
if (x > 9) {
x = x + 1; // x becomes 3
}
i32 z = add3(x, y, 10); // 15
return z;
}
